css-ui-3/caret-color-018.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Basic User Interface Test: caret-color test animation</title>
+<link rel="author" title="Manuel Rego Casasnovas" href="mailto:rego@igalia.com">
+<link rel="help" href="http://www.w3.org/TR/css3-ui/#caret-color">
+<link rel="help" href="https://www.w3.org/TR/web-animations-1/#dom-animatable-animate">
+<link rel="help" href="https://www.w3.org/TR/css3-color/#color0">
+<meta name="flags" content="dom interact animated">
```

Comment:
* `dom` isn't wrong, but it is not needed for testharness tests (also, we've decided to deprecate it in general, as you generally can infer it from the presence of scripts)
* `animated` is not needed, as the final state of this test is not animated (and we've decided to depracte this flag anyway)
* I don't think you need the `interact` flag here, nothing seems to need interaction in this test.